src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -124,48 +122,101 @@ public static void tearDown() {
      */
     @Test
     public void canSimulateApiUserFlow() {
-        // create otp user as api user
+
+        // obtain bearer token
+        String endpoint = String.format("api/secure/application/%s/authenticate?username=%s&password=%s",
+            apiUser.id,
+            apiUser.email,
+            TEMP_AUTH0_USER_PASSWORD);
+        HttpResponse<String> getTokenResponse = mockAuthenticatedPost(endpoint,
+            apiUser,
+            ""
+        );
+        System.out.println(getTokenResponse.body());
+        assertEquals(HttpStatus.OK_200, getTokenResponse.statusCode());
+
+        // create otp user as api user.
         HttpResponse<String> createUserResponse = mockAuthenticatedPost("api/secure/user",
             apiUser,
             JsonUtils.toJson(otpUser)
         );
         assertEquals(HttpStatus.OK_200, createUserResponse.statusCode());
+
+        // Create a monitored trip for an Otp user as Otp user. This will fail because the user was created by an
```

Comment:
These comments should be clarified a bit. "Create ... for an Otp user as Otp user..." should use more precise language: "Create ... for an Otp user authenticating as an Otp user..." 

Same goes for below comment.

Reply:
Updated.